### PR TITLE
refactor: use css modules

### PR DESCRIPTION
**Pull request by Zackydout101**
First changes

### CODE REVIEW

**Comment by arcanistzed** [comment]
Please provide a proper PR title using the Conventional Commits specification as described in our [Contributing docs](https://github.com/HacktheHill/.github/blob/main/CONTRIBUTING.md). I'm not able to see which issue this relates to without viewing the code.
I have also linked this PR with the corresponding GitHub issue using the "Development" section in the right sidebar. This is useful because it will automatically close the issue when this PR is merged and it provides a link between the PR and the issue.

File: src/components/Footer/Footer.module.css [hunk 1]
```diff
@@ -1,12 +1,12 @@
-.footer {
```
Comment (arcanistzed) [inline src/components/Footer/Footer.module.css:1]: Why are we removing these spaces? That doesn't align with our code formatting configuration.

File: src/components/Collaborators/MountainFlags.jsx [hunk 2]
```diff
@@ -22,7 +23,7 @@ function MountainFlags({ data }) {
 		};
 	}, []);
 
-	return <div ref={mountain} className="mountain-flags"></div>;
```
Comment (arcanistzed) [inline src/components/Collaborators/MountainFlags.jsx:25]: don't change the class name to camelCase

File: src/components/Collaborators/Collaborators.module.css [hunk 3]
```diff
@@ -1,12 +1,12 @@
-.collaborators {
+.collaborators{
 	color: var(--dark-text-color);
 	font-family: Coolvetica;
 	margin: 6% 10%;
 	font-size: 3em;
 	font-weight: normal;
 }
 
-.mountain-flags svg {
```
Comment (arcanistzed) [inline src/components/Collaborators/Collaborators.module.css:9]: don't change the class name to camelCase

File: src/components/Hero/Hero.jsx [hunk 4]
```diff
@@ -71,26 +70,26 @@ function Hero() {
...
+				className={styles["hero-img"]}
 				dangerouslySetInnerHTML={{
 					__html: hero,
 				}}
 			></div>
 			{date && (
-				<dialog className="countdown-dialog" open={popupOpen}>
-					<div className="countdown-item">
+				<dialog className={styles["countdown-dialog"]} open={popupOpen}>
+					<div className={styles["countdow-item"]}>
```
Comment (arcanistzed) [inline src/components/Hero/Hero.jsx:80]: it says `countdow` not `countdown`

File: src/components/Hero/Hero.jsx [hunk 5]
```diff
@@ -100,4 +99,4 @@ function Hero() {
 	);
 }
 
-export default Hero;
+export default Hero;
```
Comment (arcanistzed) [inline src/components/Hero/Hero.jsx:102]: no need to remove the EOF new line, follow the Prettier code formatting configuration

File: src/components/Hero/Hero.module.css [hunk 6]
```diff
@@ -101,14 +101,14 @@
 	align-items: center;
 }
 
-.countdown-item h3 {
+.countdownItem h3 {
```
Comment (arcanistzed) [inline src/components/Hero/Hero.module.css:104]: don't use camelCase

**Review by arcanistzed — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.module.css (hunk 1 above)
Comment (arcanistzed) [inline src/components/Footer/Footer.module.css:1]: revert these changes to the formatting in this file

File: src/components/Collaborators/Collaborators.module.css [hunk 7]
```diff
@@ -1,4 +1,4 @@
-.collaborators {
```
Comment (arcanistzed) [inline src/components/Collaborators/Collaborators.module.css:1]: revert these changes to the formatting in this file

File: src/components/App/global.css [hunk 8]
```diff
@@ -1,4 +1,4 @@
-:root {
```
Comment (arcanistzed) [inline src/components/App/global.css:1]: revert these changes to the formatting in this file

File: src/components/Hero/Hero.module.css [hunk 9]
```diff
@@ -1,4 +1,4 @@
-.hero {
```
Comment (arcanistzed) [inline src/components/Hero/Hero.module.css:1]: revert these changes to the formatting

File: src/components/Hero/Hero.module.css [hunk 10]
```diff
@@ -108,7 +108,7 @@
 }
 
 .countdown-item h4 {
-	color: var(--countdown-item-color);
```
Comment (arcanistzed) [inline src/components/Hero/Hero.module.css:111]: revert these changes to the formatting

File: src/components/Notification/Notification.jsx [hunk 11]
```diff
@@ -19,13 +20,13 @@ function Notification() {
...
 		<div
 			id="notification"
-			className="notification"
+			className={ styles.notification}
 			style={{
 				transform: visible ? "translateY(0)" : "translateY(100%) scaleY(0)",
 			}}
 			aria-hidden={!visible}
 		>
-			<div className="notification-content">
```
Comment (arcanistzed) [inline src/components/Notification/Notification.jsx:28]: revert these changes to the formatting

File: src/components/Notification/Notification.module.css [hunk 12]
```diff
@@ -14,7 +14,7 @@
 	transform: translateY(100%) scaleY(0);
 }
 
-.notification-content {
```
Comment (arcanistzed) [inline src/components/Notification/Notification.module.css:17]: revert these changes to the formatting

File: src/components/Register/Register.module.css [hunk 13]
```diff
@@ -0,0 +1,16 @@
+* {
```
Comment (arcanistzed) [inline src/components/Register/Register.module.css:1]: we don't need this file??

File: src/components/Sponsors/Sponsors.module.css [hunk 14]
```diff
@@ -18,7 +18,7 @@
 	gap: 5rem;
 }
 
-.sponsors-row {
```
Comment (arcanistzed) [inline src/components/Sponsors/Sponsors.module.css:21]: revert these changes to the formatting

File: src/components/Schedule/Schedule.module.css [hunk 15]
```diff
@@ -25,7 +37,7 @@
 	background-size: 10%, 8%, 2%;
 }
 
-.schedule-title {
```
Comment (arcanistzed) [inline src/components/Schedule/Schedule.module.css:28]: revert these changes to the formatting in this file

**Review by arcanistzed — CHANGES_REQUESTED** [review]

**Comment by xsachax** [comment]
should be good to merge, could use a last look-over

File: src/components/Sponsors/Sponsors.jsx [hunk 16]
```diff
@@ -66,26 +66,28 @@ function Sponsors() {
...
+			<section id="sponsors" className={styles["sponsors"]}>
 				<h1>{t("sponsors.title")}</h1>
 				<div>
-					<p className="sponsors-text">{t("sponsors.p")}</p>
+					<p className={styles["sponsors-text"]}>{t("sponsors.p")}</p>
 					<Button href="/assets/Hack-the-Hill-Sponsorship.pdf">{t("sponsors.button")}</Button>
 				</div>
 
-				<div className="sponsor-icons">
+				<div className={styles["sponsors-col"]}>
```
Comment (arcanistzed) [inline src/components/Sponsors/Sponsors.jsx:76]: name should be `sponsor-icons`

File: src/components/Sponsors/Sponsors.jsx [hunk 17]
```diff
@@ -66,26 +66,28 @@ function Sponsors() {
...
 					{Object.values(data.sponsors).map((tier, i) => (
-						<div key={i} className="sponsor-icons-row">
+						<div key={i} className={styles["sponsors-row"]}>
```
Comment (arcanistzed) [inline src/components/Sponsors/Sponsors.jsx:78]: should be `sponsor-icons-row`

File: src/components/Sponsors/Sponsors.jsx [hunk 18]
```diff
@@ -66,26 +66,28 @@ function Sponsors() {
...
 							{tier.map((sponsor, j) => (
 								<a
 									key={j}
 									href={sponsor.href}
 									target="_blank"
 									rel="noreferrer"
-									className="sponsor-icon-box"
+									className={styles["sponsors-icon-box"]}
```
Comment (arcanistzed) [inline src/components/Sponsors/Sponsors.jsx:85]: should be `sponsor-icon-box`

File: src/components/Sponsors/Sponsors.jsx [hunk 19]
```diff
@@ -96,13 +98,19 @@ function Sponsors() {
 				</div>
 			</section>
 
-			<section id="community" className="sponsors">
```
Comment (arcanistzed) [inline src/components/Sponsors/Sponsors.jsx:99]: shouldn't be removed

**Review by arcanistzed — CHANGES_REQUESTED** [review]

**Review by arcanistzed — APPROVED** [review]

**Pull request merged**